platforms/platforms.go

```diff
@@ -270,12 +288,22 @@ func MustParse(specifier string) specs.Platform {
 }
 
 // Format returns a string specifier from the provided platform specification.
-func Format(platform specs.Platform) string {
-	if platform.OS == "" {
+func Format(ocispecPlatform specs.Platform) string {
```

Comment:
Would we need to do a `FormatV2` or something that we aren't transparently changing this conversion?

Reply:
We would need the host OS version filled in for windows all the time.